spec/lib/miq_apache/control_spec.rb

```diff
@@ -19,14 +19,14 @@
     allow(File).to receive(:exist?).and_return(false)
     expect(Dir).to receive(:mkdir).with(File.dirname(MiqApache::Control::APACHE_CONTROL_LOG))
     allow(MiqUtil).to receive(:runcmd)
-    MiqApache::Control.run_apache_cmd("start")
```

Comment:
@jrafanie @szemek I do not understand how this spec change is related to the code changes above

Reply:
@chessbyte It wasn't private before so you were allowed to call it.  Now that it's private, you need to either use `send` or call the "caller" instead.